src/lib.rs

```diff
@@ -156,7 +156,7 @@ impl Fernet {
         )
         .unwrap();
 
-        let mut result = Vec::new();
+        let mut result = vec![];
         result.push(0x80);
```

Comment:
I assume this was the [`vec_init_then_push`](https://rust-lang.github.io/rust-clippy/master/#vec_init_then_push) lint. Following that lint as specified would have replaced this code with
```rust
let mut result = vec![0x80];
```